scholly/base/src/global/data.js

```diff
@@ -46,7 +46,6 @@ if (!env.REACT_APP_DEBUG) {
     ? JSON.parse(env.REACT_APP_WEB_PAGES)
     : [];
 } else {
-  console.log('Running in DEBUG mode, hence using fake Team data');
```

Comment:
Actually, would prefer to leave the `console.log` here as it's useful to know that we're running in Debug mode, or else it can be confusing whether we're using real or fake data. Maybe  we could also modify the log to read `..., hence using fake data`, as we're not only using fake Team data.

Reply:
Sure, added this line back.